### DISCUSSION TITLE
Lost my dashboard

### DISCUSSION THREAD

**Discussion by mathgoy** [revenz/Fenrus#180] [Q&A] [answered]
Hi guys, 

I amusing Fenrus for months on my unraid and, since the last update, I get the default dashboard when I login.
The funny thing is I can still see the JSON config file in the docker folder.

What can I do to "reload" that dashboard I made ?

Thanks

**Comment by eqjunkie829** [comment]
I changed to tag to revenz/fenrus:preview to revert to old version. I didnt really have the time to research the update this morning and ill probably upgrade later.

**Comment by revenz** [comment, marked as answer]
there are upgrade instructions to migrate your config to the new fenrus. or if you are happy with the old stick with tthat.  dockerhub will likely automatically remove the image at some point though, so you may get stuck if you never upgrade
